ui: node name label update

vladlos opened the pull request:
on chart legends, node id is cut off, so we put it before ip address as it is more important information

Resolves: #45105

Release note(ui): updated node label in chart legends to make node id visible.

before:
![Screenshot 2020-06-19 at 17 29 29](https://user-images.githubusercontent.com/12850886/85143548-8cf2b480-b252-11ea-9b69-2152f88335d8.png)

after:
![Screenshot 2020-06-19 at 17 30 38](https://user-images.githubusercontent.com/12850886/85143589-9bd96700-b252-11ea-9680-f9d28f9f7e5a.png)

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/50434)
<!-- Reviewable:end -->

koorosh commented:
@elkmaster , should we put Node id into parentheses if it is placed at the very beginning?
@Annebirzin , do you have any suggestions on formatting a legend with swapped Node Id and address?

Annebirzin commented:
@koorosh I think we can put the node id in parentheses as you have it, so it would look like `(n1)localhost 2625...`

koorosh reviewed (approved):
LGTM

vladlos committed:
ui: node name label update

on chart legends, node id is cut off, so we put it before ip address as it is more important information

Resolves: #45105

Release note(ui): updated node label in chart legends to make node id visible.

nathanstilwell reviewed (approved):
LGTM

elkmaster commented:
bors r+

rafiss commented:
FYI it looks like this PR did not pass the bors merging checks because the license agreement was not signed.

dhartunian commented:
bors r+
